Source/WebCore/rendering/StyledMarkedText.cpp

```diff
@@ -40,7 +40,7 @@ static void computeStyleForPseudoElementStyle(StyledMarkedText::Style& style, co
     if (!pseudoElementStyle)
         return;
 
-    style.backgroundColor = pseudoElementStyle->colorResolvingCurrentColor(pseudoElementStyle->backgroundColor());
+    style.backgroundColor = pseudoElementStyle->visitedDependentColorWithColorFilter(CSSPropertyBackgroundColor, paintInfo.paintBehavior);
```

Comment:
Is this web exposed through `getComputedStyle(..., "::target-text")` ? I'm guessing not, but it would be nice to get a testcase to confirm.

Reply:
No, this is already covered by `css/css-pseudo/highlight-pseudos-visited-computed-001.html`.

I've updated the commit message to mention that.